src/test/java/uk/gov/pay/adminusers/service/ServiceUpdaterTest.java

```diff
@@ -51,6 +52,7 @@ public void before() throws Exception {
     public void shouldUpdateNameSuccessfully() {
         String serviceId = randomUuid();
         String nameToUpdate = "new-name";
+        Map<SupportedLanguage, ServiceNameEntity> languageMap = mock(HashMap.class);
```

Comment:
I wouldn’t bother mocking the map: just use a real one and verify its actual contents